src/utils/translation-banner.ts

```diff
@@ -0,0 +1,12 @@
+import type { LocaleObject } from '@nuxtjs/i18n'
+
+const BASE_URL = 'https://translate.wordpress.org/projects/meta/openverse/'
+
+export const needsTranslationBanner = (locale: LocaleObject) => {
+  if (!locale || locale.code === 'en') return false
+
+  return (locale.translated ?? 100) <= 90
```

Comment:
I might prefer if 90 was captured in a variable called like `MINIMUM_TRANSLATION_PERCENTAGE` or if there was a comment explaining what it represents. I don't love having an arbitrary number like that without context.